modules/api/php/endpoints/project/project.class.inc

```diff
@@ -113,6 +113,9 @@ class Project extends Endpoint implements \LORIS\Middleware\ETagCalculator
         case 'visits':
             $handler = new Visits($this->_project);
             break;
+        case 'recordings':
```

Comment:
looks like I did not add one of the files to Github for the project/recordings endpoint... Will do that tomorrow as this is most probably still on my other computer...